packages/auth/src/authClients/supertokens.ts

```diff
@@ -0,0 +1,44 @@
+import Sessions from 'supertokens-auth-react/recipe/session'
```

Comment:
See comments and a suggested change to move importing `Sessions` into each auth client method.